### ISSUE TITLE
Support for Custom Fonts? Custom Training?

### ISSUE THREAD

**Issue by rooseveltrp** [JaidedAI/EasyOCR#261]
Hi,

I am testing out EasyOCR and it's really good! EasyOCR passed all my use cases but I noticed its accuracy drops when a custom font is used.

For example, VICTORY is `VGIORY` and Royale seems to be `OYJIE`.

Can we train/use EasyOCR with a custom font?

Great work team! It's pretty amazing already! 

Thanks

![image](https://user-images.githubusercontent.com/4078130/93413519-4bbf7000-f854-11ea-80b4-60ac21701d19.png)

**Comment by rkcosmos** [comment]
We plan to add support for data generation (using custom font) and training script soon (hopefully within this year).

**Comment by azzarello** [comment]
Any update on this? I am running into a similar issue with a project where I want to grab url's from the url box on a Twitch stream, and I am sure if I used only training data from the default chrome font, it would be a lot more accurate.

**Comment by rakesh4real** [comment]
Hey, @rkcosmos can you please post the related PRs and issues for this? Will contribute!

**Comment by rkcosmos** [comment]
Please see [this note](https://github.com/JaidedAI/EasyOCR/blob/master/custom_model.md).
